nixos/modules/system/activation/top-level.nix

```diff
@@ -224,6 +214,36 @@ in
       '';
     };
 
+    system.build = lib.tmpFixupSubmoduleBoundary {
+      installBootLoader = mkOption {
+        internal = true;
+        default = "echo 'Warning: do not know how to make this configuration bootable; please enable a boot loader.' 1>&2; true";
```

Comment:
I've added a comment:
```
        # "; true" => make the `$out` argument from switch-to-configuration.pl
        #             go to `true` instead of `echo`, hiding the useless path
        #             from the log.
        default = "echo 'Warning: do not know how to make this configuration bootable; please enable a boot loader.' 1>&2; true";
```

Reply:
Ah, thanks, didn't realise it was called with `$out`